WordPress/Classes/Services/PostService.m

```diff
@@ -128,9 +128,15 @@ - (void)getPostWithID:(NSNumber *)postID
                           }
                           if (remotePost) {
                               AbstractPost *post = [self findPostWithID:postID inBlog:blog];
+                              
                               if (!post) {
-                                  post = [self createPostForBlog:blog];
+                                  if ([remotePost.type isEqualToString:PostServiceTypePage]) {
```

Comment:
Just a note that there's an identical snippet of code in this class, so we could place this logic inside a method.

That said, because this is a fix for 18.3 which is soon going out, and because I don't want to risk breaking any existing logic, I'm purposedly only changing this part of the code right now.

I'll push another PR unifying both snippets into a single method against `develop`.

Reply:
Sounds good, I agree with your thinking!